libs/s25main/gameData/BuildingConsts.h

```diff
@@ -81,31 +81,34 @@ extern const helpers::MultiEnumArray<SmokeConst, Nation, BuildingType> BUILDING_
+     {-2, -15}, {2, -13},  {-5, -16}, {-5, -15}, {0, 0},    {22, 6},   {0, 0},    {4, -16},   {9, -12},   {7, -10},
      {28, -18}, {-6, -16}, {-3, -15}, {21, -11}, {14, -7},  {-14, -9}, {11, -9},  {-5, -24},  {-18, -34}, {0, 0},
      {0, 0},    {3, -13},  {13, -13}, {-4, -20}, {7, -14},  {15, -4},  {0, 0},    {-22, -16}, {-14, -8},  {0, 0}}},
    // Japaner
-   {{{0, 0},    {12, -10}, {13, -10},  {0, 0},    {25, -22}, {-26, -13}, {-8, -8},   {9, -10},   {0, 0},     {10, -14},
-     {20, -2},  {12, -14}, {14, -13},  {13, -14}, {0, 0},    {0, 0},     {0, 0},     {-8, -6},   {0, 0},     {-13, -7},
+   {{{0, 0},    {12, -10}, {13, -10},  {13, -1},  {25, -22}, {-26, -13}, {-8, -8},   {9, -10},   {-6, 2},    {10, -14},
+     {20, -2},  {12, -14}, {14, -13},  {13, -14}, {0, 0},    {4, -18},   {0, 0},     {-8, -6},   {0, 0},     {-13, -7},
      {14, -11}, {15, -10}, {-13, -7},  {16, -11}, {21, -3},  {-6, -2},   {14, -14},  {-26, -17}, {30, -25},  {0, 0},
      {0, 0},    {0, -22},  {-30, -13}, {35, -20}, {13, -34}, {19, -15},  {-22, -10}, {37, -13},  {-15, -36}, {0, 0}}},
    // Römer
-   {{{0, 0},   {15, -3}, {14, -2},  {0, 0},    {0, 0},    {36, -12}, {-8, -8},  {12, -19},  {0, 0},     {20, -39},
-     {15, -2}, {26, 4},  {21, 4},   {21, 4},   {0, 0},    {0, 0},    {0, 0},    {-29, -10}, {0, 0},     {-4, 4},
+   {{{0, 0},   {15, -3}, {14, -2},  {13, -7},  {0, 0},    {36, -12}, {-8, -8},  {12, -19},  {-6, -2},   {20, -39},
+     {15, -2}, {26, 4},  {21, 4},   {21, 4},   {0, 0},    {-15, -8}, {0, 0},    {-29, -10}, {0, 0},     {-4, 4},
      {7, -20}, {3, -13}, {0, 0},    {22, -8},  {18, -11}, {6, -14},  {23, -10}, {-28, -12}, {35, -27},  {0, 0},
      {0, 0},   {2, -21}, {15, -13}, {11, -30}, {23, -5},  {10, -9},  {0, -25},  {41, -43},  {-34, -19}, {0, 0}}},
    // Wikinger
-   {{{0, 0},  {-5, 0},  {-5, 0},   {0, 0},    {-7, -5},  {-7, -32}, {-10, -1}, {-4, 5},    {0, 0},    {17, -10},
-     {20, 2}, {20, 0},  {20, -2},  {20, 0},   {0, 0},    {0, 0},    {13, -5},  {15, -6},   {-7, 2},   {17, -8},
+   {{{0, 0},  {-5, 0},  {-5, 0},   {-7, -4},  {-7, -5},  {-7, -32}, {-10, -1}, {-4, 5},    {-7, 1},   {17, -10},
+     {20, 2}, {20, 0},  {20, -2},  {20, 0},   {0, 0},    {-15, -8}, {13, -5},  {15, -6},   {-7, 2},   {17, -8},
      {-5, 0}, {-3, 2},  {-6, 0},   {-7, 0},   {-32, -2}, {-13, -3}, {-8, -2},  {-22, -18}, {-25, -8}, {0, 0},
      {0, 0},  {-8, -2}, {-17, -4}, {28, -16}, {-1, 0},   {8, -9},   {16, -15}, {-2, -25},  {-29, -9}, {0, 0}}},
    // Babylonier
-   {{{0, 0},    {19, -6},  {19, -20}, {0, 0},    {5, -15},  {-32, -20}, {4, -16}, {10, -18},  {0, 0},    {20, -10},
-     {15, -10}, {15, -10}, {15, -10}, {15, -10}, {0, 0},    {0, 0},     {13, -5}, {-5, -13},  {15, -20}, {15, -1},
+   {{{0, 0},    {19, -6},  {19, -20}, {-7, -3},  {5, -15},  {-32, -20}, {4, -16}, {10, -18},  {3, -18},  {20, -10},
+     {15, -10}, {15, -10}, {15, -10}, {15, -10}, {0, 0},    {5, -4},    {13, -5}, {-5, -13},  {15, -20}, {15, -1},
```

Comment:
And Babylonian Leatherworks X sign should be: {15,-31}

Reply:
done